### ISSUE TITLE
local build instructions

### ISSUE THREAD

**Issue by alxv2016** [Xpndable/DotUI#11]
Hi, are you able to provide some instructions to how I can run a build on my local machine?  Much appreciated!

**Comment by alxv2016** [comment]
Managed to figure out how to build on my local machine but build keeps erroring out at line 57 in makefile with picoarch...any ideas?

**Comment by njbraun** [comment]
@alxv2016 Same exact issue, fails with:

```
libretro/libretro.c: In function 'retro_get_system_info':
libretro/libretro.c:980:26: error: 'PokeMini_Version' undeclared (first use in this function); did you mean 'PokeMini_Reset'?
  info->library_version = PokeMini_Version _GIT_VERSION;
                          ^~~~~~~~~~~~~~~~
                          PokeMini_Reset
libretro/libretro.c:980:26: note: each undeclared identifier is reported only once for each function it appears in
libretro/libretro.c:980:42: error: expected ';' before string constant
  info->library_version = PokeMini_Version _GIT_VERSION;
                                          ^
                                          ;
```

**Comment by alxv2016** [comment]
> @alxv2016 Same exact issue, fails with:
> 
> ```
> libretro/libretro.c: In function 'retro_get_system_info':
> libretro/libretro.c:980:26: error: 'PokeMini_Version' undeclared (first use in this function); did you mean 'PokeMini_Reset'?
>   info->library_version = PokeMini_Version _GIT_VERSION;
>                           ^~~~~~~~~~~~~~~~
>                           PokeMini_Reset
> libretro/libretro.c:980:26: note: each undeclared identifier is reported only once for each function it appears in
> libretro/libretro.c:980:42: error: expected ';' before string constant
>   info->library_version = PokeMini_Version _GIT_VERSION;
>                                           ^
>                                           ;
> ```

I just figured out it caused by PokeMini in the picoarch. If you comment out references to PokeMini cores it will build without any issues.

**Comment by Xpndable** [comment]
this is the pokemini issue I ran into, to fix that I added this line under line 77 in the makefile for picoarch
`pokemini_REVISION = 684e7ea0950f4df48cd1fbf1160e6af3c262c9f0`
that'll pin it to the last known good build point for this version of the picoarch makefile

**Comment by Xpndable** [comment]
Also, note that the port of vvvvvv was commented out of the DotUI makefile in this repository, and will be resolved in line with this issue #15
